pyteal/ast/abi/util_test.py

```diff
@@ -593,3 +594,11 @@ def test_abi_type_translation(algosdk_abi, abi_string, pyteal_abi_ts, pyteal_abi
     )
     assert algosdk_abi == abi.algosdk_from_type_spec(pyteal_abi_ts)
     assert algosdk_abi == abi.algosdk_from_annotation(pyteal_abi)
+
+
+def test_sdk_abi_translation():
+    for case in ABI_TRANSLATION_TEST_CASES:
+        # Errors are strings in the 0th element
+        if type(case[0]) == str:
```

Comment:
```suggestion
        if type(case[0]) is str:
```
Types are singleton's so prefer `is`